widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java

```diff
@@ -1150,7 +1159,18 @@ public void newEngagementLoaded(OmnicoreEngagement engagement) {
     public void engagementEnded() {
         Logger.d(TAG, "engagementEnded");
         stop();
-        dialogController.showEngagementEndedDialog();
```

Comment:
Survey is optional? What will happen if engagement should end without survey

Reply:
If survey is null dialogController.showEngagementEndedDialog() called in the newSurveyLoaded(Survey).